bciers/apps/registration/app/components/operations/registration/RegistrationSubmissionForm.tsx

```diff
@@ -63,19 +63,10 @@ const RegistrationSubmissionForm = ({
         <MultiStepBase
           allowBackNavigation
           baseUrl={`/register-an-operation/${operation}`}
```

Comment:
Is `baseUrl` being used for the final page?

Reply:
Nope!